pkg/plugin/queries.go

```diff
@@ -35,18 +35,27 @@ const (
 func buildFrames(responses []internal.FrameData, frameType frameType, frames []*data.Frame) []*data.Frame {
 	frameMap := make(map[string]*data.Frame)
 
+	var frameToAppendTo *data.Frame
 	for _, frameDataItem := range responses {
 		timestamp, err := frameDataItem.GetTimestamp()
 		if err != nil {
 			log.DefaultLogger.Error("error while parsing time %w", err)
 			continue
 		}
-		key := frameDataItem.GetKey()
-		frameToAppendTo, ok := frameMap[key]
-		if !ok {
-			frameDefinition := getFrameDefinitionFunction(frameType, frameDataItem)()
+		frameDefinition := getFrameDefinitionFunction(frameType, frameDataItem)()
+		// For table Wide frames, we always want to append to the one single frame in order
+		if frameType == TableFrameType && frameToAppendTo == nil {
 			frameToAppendTo = &frameDefinition
-			frameMap[key] = frameToAppendTo
+			frameMap["fixed-table"] = frameToAppendTo
+		} else if frameType == GraphFrameType {
+			key := frameDataItem.GetKey()
+
+			var ok bool
+			frameToAppendTo, ok = frameMap[key]
```

Comment:
You can probably drop the var
```suggestion
			frameToAppendTo, ok := frameMap[key]
```

Reply:
Can't, frameToAppend to has to be declared outside.